From b0aa0ec0e8ce36b9b537edf85f2c25b73d4c36d2 Mon Sep 17 00:00:00 2001
From: vdanix <vishwanathx.dani@intel.com>
Date: Thu, 29 Oct 2020 13:34:47 +0530
Subject: [PATCH] CIV: Gpu service rule added in sepolicy

Added gpu service rule to get Vulkan device
array list to pass GtsGraphicsHostTestCases

Change-Id: I837448a332f34efa4d69b7af6e324f4d8bfa9b36
Tracked-On: OAM-94431
Signed-off-by: vdanix <vishwanathx.dani@intel.com>

diff --git a/vendor/gpusevice.te b/vendor/gpusevice.te
new file mode 100644
index 0000000..96ad1a7
--- /dev/null
+++ b/vendor/gpusevice.te
@@ -0,0 +1,3 @@
+#============= gpuservice ==============
+allow gpuservice gpu_device:dir { open read search };
+allow gpuservice sysfs_app_readable:file { open read };
--
2.29.0

